subsys/nrf_security/src/drivers/cracen/Kconfig

```diff
@@ -16,7 +16,8 @@ if PSA_CRYPTO_DRIVER_CRACEN
 
 config CRACEN_LOAD_MICROCODE
 	bool "Load CRACEN microcode"
-	depends on (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX
+	depends on (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX || \
+		(SOC_SERIES_NRF92X && (SOC_NRF9280_CPUSEC || SOC_NRF9230_ENGB_CPUSEC))
```

Comment:
this should also be reworked to have a new symbol:
```
config SUPPORTS_CRACEN
	default y if (SOC_SERIES_NRF54LX && !SOC_NRF54L20) || SOC_SERIES_NRF54HX
```
this would be the whole definition here, this symbol would then become:
```
config CRACEN_LOAD_MICROCODE
	bool "Load CRACEN microcode"
	depends on SUPPORTS_CRACEN
	default y
```
then in your out of tree soc for cpusec inside the `Kconfig` file:
```
config SUPPORTS_CRACEN
	default y if (SOC_SERIES_NRF92X && (SOC_NRF9280_CPUSEC || SOC_NRF9230_ENGB_CPUSEC))
```
then you have everything properly isolated with no dummy symbols here, and no leakage of information